ffmpeg-7.1.yaml

```diff
@@ -34,8 +34,11 @@ environment:
       - dav1d-dev
       - expat-dev
       - fontconfig-dev
+      - freetype-dev
+      - fribidi-dev
       - harfbuzz-dev
       - lame-dev
+      - libass
```

Comment:
huh, seems like `libass` needs to `split/dev` a subpackage

Reply:
Well noticed. That points to a bigger problem, that we're likely statically linking it. if we are, then there's nothing telling us to rebuild ffmpeg if a CVE is found in libass. Looks like we need to:

1. Add `-Ddefault_library=both` as an `opts` to the `meson/configure` pipeline.
2. Add a -dev package with `split/dev`
3. We should bump any rdeps that build against `libass` to make sure they are linking dynamically... but we don't appear to have any.
4. While we're at it, the build says that we have asm optimizations disabled. Can you add `nasm` as a build-dependency?